jetbrains-core/resources/META-INF/ext-python.xml

```diff
@@ -7,11 +7,11 @@
     </extensions>
 
     <extensions defaultExtensionNs="aws.toolkit.lambda">
-        <runtimeGroup runtimeGroup="PYTHON" implementation="software.aws.toolkits.jetbrains.services.lambda.python.PythonRuntimeGroup"/>
-        <builder runtimeGroup="PYTHON" implementation="software.aws.toolkits.jetbrains.services.lambda.python.PythonLambdaBuilder"/>
-        <handlerResolver runtimeGroup="PYTHON" implementation="software.aws.toolkits.jetbrains.services.lambda.python.PythonLambdaHandlerResolver"/>
-        <sam.debugSupport runtimeGroup="PYTHON" implementation="software.aws.toolkits.jetbrains.services.lambda.python.PythonSamDebugSupport"/>
-        <sam.projectWizard runtimeGroup="PYTHON" implementation="software.aws.toolkits.jetbrains.services.lambda.python.PythonSamProjectWizard"/>
+        <runtimeGroup implementation="software.aws.toolkits.jetbrains.services.lambda.python.PythonRuntimeGroup"/>
+        <builder id="PYTHON" implementationClass="software.aws.toolkits.jetbrains.services.lambda.python.PythonLambdaBuilder"/>
```

Comment:
Why do these need ID? Can the be children of runtimeGroup? If we need them across multiple files can we merge them together? It's weird that above the runtime group defines the ID, but you'd need to go to the runtime group to know that. Then it is used below but they don't necessarily look connected.

Reply:
I can also make runtimeGroup's ID be here too for consistency sake

Reply:
Thinking about this more I'm not sure what is preferable, I think this is fine